README.md

```diff
@@ -468,6 +471,87 @@ mailer.set_attachments(attachments, mail_body)
 mailer.send(mail_body)
 ```
 
+## Bulk Email
+
+### Send bulk email
+
+```python
+from mailersend import emails
+
+api_key = "API key here"
+
+mailer = mailersend.NewEmail(api_key)
+
+mail_list = [
+  {
+    "from": {
+      "email": "hello@mailersend.com",
```

Comment:
can we keep example emails and names the same throughout 

your@domain.com etc

Reply:
I copied that over from developers.mailersend and didn't bother to check other sdks 😄  will change